incremental-font-transfer/src/patchmap.rs

```diff
@@ -734,12 +734,26 @@ impl Ord for IntersectionInfo {
     }
 }
 
+/// Indicates a malformed URI template was encountered.
+///
+/// More info: <https://datatracker.ietf.org/doc/html/rfc6570#section-3>
+#[derive(Debug, PartialEq, Eq)]
+pub struct UriTemplateError;
```

Comment:
what error does the new crate that we're using return? do we need our own error type?

Reply:
I want to avoid exposing details of the internal crate used in our api.